app/assets/_dev/scss/ailab/_ai_lab_home.scss

```diff
@@ -0,0 +1,58 @@
+.template-homepage {
```

Comment:
Can we make this class ai lab specific, to prevent it potentially leaking out to other pages?

Reply:
Good spot, have just pushed an update